packages/neo4j-driver/test/internal/connection-channel.test.js

```diff
@@ -278,31 +273,32 @@ describe('#integration ChannelConnection', () => {
     testQueueingOfObserversWithBrokenConnection(resetAction, done)
   })
 
-  it('should reset and flush when SUCCESS received', async done => {
-    connection = await createConnection(`bolt://${sharedNeo4j.hostnameWithBoltPort}`)
+  it('should reset and flush when SUCCESS received', done => {
+    createConnection(`bolt://${sharedNeo4j.hostnameWithBoltPort}`).then(conn => {
```

Comment:
there is no await here. The async/await was removed from this test since we need to work with promises to be able to call `resetAndFlush` and simulate some protocol messages before the resolution of the promise. There are other tests doing something similar. 

The mix of `async/await` with the callback `done` is not supported anymore in newer versions of karma, the is actually good.